__tests__/Unit/Components/Tasks/TaskLevelMap.test.tsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
-import { fireEvent, screen } from '@testing-library/react';
+import { fireEvent, screen, render } from '@testing-library/react';
 import { TaskLevelMap } from '@/components/tasks/card/TaskLevelMap';
-import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
 import taskItem from '@/interfaces/taskItem.type';
 
 import { renderWithProviders } from '@/test-utils/renderWithProvider';
 
+jest.mock('@/hooks/useUserData', () => {
+    return () => ({
+        data: {
+            roles: {
+                admin: true,
+                super_user: false,
+            },
+        },
+        isUserAuthorized: true,
+        isSuccess: true,
+    });
+});
+
```

Comment:
This can be moved to `test/utils` and imported from there. In this test __tests__/Unit/Components/Tasks/Card.test.tsx as well the same code is being repeated. 
If in the future you need to use this in other places, we can avoid repetition by adding it to utils.

Reply:
I will be addressing this issue in this ticket

https://github.com/Real-Dev-Squad/website-status/issues/670